isaric/parsers/isaric-ccpuk.toml

```diff
@@ -0,0 +1,450 @@
+[adtl]
+name = "isaric-ccpuk"
+description = "ISARIC CCPUK study"
+
+[adtl.tables]
+study = { kind = "constant" }
+subject = { kind = "groupBy", groupBy = "subject_id", aggregation = "lastNotNull" }
+visit = { kind = "groupBy", groupBy = "visit_id", aggregation = "lastNotNull" }
+
+[adtl.defs."Y/N/NK".values]
+1 = true
+2 = false
+
+[study]
+id = "isaric-ccpuk"
+date = "2021-01-01"
+country_iso3 = "GBR"
+
+## SUBJECT
+
+[subject]
+study_id = "isaric-ccpuk"
+country_iso3 = "GBR"
+pathogen = "COVID-19"
+
+[visit]
```

Comment:
Should these two lines be here? are they misplaced?

Reply:
I agree, I'd put these with the rest of the visit table

Reply:
Thanks, fixed!